thorlcr/activities/nsplitter/thnsplitterslave.cpp

```diff
@@ -210,9 +210,13 @@ class NSplitterSlaveActivity : public CSlaveActivity, implements ISharedSmartBuf
     {
         // NB: called by 1st output to start()
         CriticalBlock block(prepareInputLock);
+        if (writeAheadException.get()) // rethrow any previously caught exception
```

Comment:
trivial: we don't normally call .get() on a Owned/Linked object (in the same way the assertex you added doesn't, admittedly there are some places that do).
We normally rely on the overloaded operator to get the pointer.